tower-http/Cargo.toml

```diff
@@ -12,6 +12,9 @@ categories = ["asynchronous", "network-programming", "web-programming"]
 keywords = ["io", "async", "futures", "service", "http"]
 rust-version = "1.60"
 
+[package.metadata]
+msrv = "1.60.0"
```

Comment:
What tooling uses this? A few lines above, `rust-version` declares the same thing.

Reply:
Oh wow, can't believe I missed that right above, https://github.com/foresterre/cargo-msrv on `cargo msrv verify` checks the [package.metadata] maybe it's only that.